src/Serilog/Serilog.csproj

```diff
@@ -3,8 +3,12 @@
     <Description>Simple .NET logging with fully-structured events</Description>
     <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
+    <!-- .NET Framework version targeting is frozen at these two TFMs. -->
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net471</TargetFrameworks>
-    <TargetFrameworks>$(TargetFrameworks);netstandard2.1;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
+    <!-- Policy is to trim TFM-specific builds to `netstandard2.0`, `net6.0`,
```

Comment:
reying to shoehorn it onto two lines:
```suggestion
    <!-- Policy is to trim TFM-specific builds when upping Serilog major version number to:
    `netstandard2.0`, `net6.0`, active LTS versions and (optionally) the latest RTM -->
```

Reply:
Either way, this all looks pretty complete and concise, and I'd say it's ready to merge.
It could be argued that a note in a wiki or the README might make sense, but I think #1970 (which this PR should resolve) and the text above should be enough - we just need a snippet at the ready saying
"discussing the TFM list has been flogged to death in #1970; policy is to drive the TFM list based on how Serilog's src/Serilog.csproj for the foreseeable" :D

Reply:
👍 diminishing returns on this, we can still revisit further along in the process :-)